docs/guides/alpine/_include/cleanup.rst

```diff
@@ -0,0 +1,21 @@
+Prepare for first boot
+----------------------
+
+Exit the chroot, unmount everything
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block::
+
+  exit
+
+.. code-block::
+
+   cut -f2 -d\  /proc/mounts | grep ^/mnt | tac | while read i; do umount -l $i; done
```

Comment:
It might be more legible to quote the space delimiter rather than quote it:
```suggestion
   cut -f2 -d" "  /proc/mounts | grep ^/mnt | tac | while read i; do umount -l $i; done
```

Reply:
good call, done!